lib/pushmi_pullyu/cli.rb

```diff
@@ -26,12 +27,19 @@ def parse(argv = ARGV)
   end
 
   def run
-    if options[:daemonize]
-      start_server_as_daemon
-    else
-      # If we're running in the foreground sync the output.
-      $stdout.sync = $stderr.sync = true
-      start_server
+    configure_rollbar
+    begin
+      if options[:daemonize]
+        start_server_as_daemon
+      else
+        # If we're running in the foreground sync the output.
+        $stdout.sync = $stderr.sync = true
+        start_server
+      end
+    # rubocop:disable Lint/RescueException
```

Comment:
Have an example off your head that wouldn't be captured by StandardError? I'm in agreeing with leaving it as is. Even though I hate inline rubocop comments.